.gitignore

```diff
@@ -38,3 +38,5 @@ docker-compose.override.yml
 .DS_Store
 .byebug_history
 .buildconfig
+osem_development
+osem_test
```

Comment:
If we have the prefix then this is not needed right?

Reply:
Does it hurt to exclude them?